.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules
 .DS_Store
+package-lock.json
```

Comment:
If we stop tracking this file don't we also want to delete it from the repo?

Reply:
👍